README.md

```diff
@@ -4,4 +4,99 @@ This repository contains plugins for
 [Yosys](https://github.com/YosysHQ/yosys.git) developed as
 [part of the SymbiFlow project](https://symbiflow.github.io).
 
+## List of plugins
+1. [Design introspection](#design_introspection)
+2. [Fasm](#fasm)
+3. [Get count](#get_count)
+4. [Integrate inverters](#integrate_inv)
+5. [Parameters](#parameters) 
+6. [QL IOBs](#ql_iob)
+7. [SDC](#sdc)
+8. [Selection](#selection)
+9. [XDC](#xdc)
 
+## Summary
+
+### Design introspection plugin <a name="design_introspection"></a>
+
+Adds several commands that allow for collecting information about cells, nets, pins and ports in the design.
+
+Following commands are added with the plugin:
+* get_cells
+* get_nets
+* get_pins
+* get_ports
+
+### Fasm plugin <a name="fasm"></a>
+
+Writes out the design's [fasm features](https://symbiflow.readthedocs.io/en/latest/fasm/docs/specification.html) based on the parameter annotations on a design cell.
+
+The plugin adds the following command:
+* write_fasm
+
+### Get count plugin <a name="get_count"></a>
```

Comment:
I wonder if this plugin should be part of the Design introspection plugin above?

Reply:
Opened #79